### DISCUSSION TITLE
Multi server support for homes, warps and player teleport.

### DISCUSSION THREAD

**Discussion by DevBD1** [Zrips/CMI#2892] [Feature Requests]
CMI already has bungee chat feature, I ask for expanding this.
I am setting up a gamemode with multiple sub-servers. Let's say we have 3 servers; A, B and C. A is the hub with spawn area, B and C they are open worlds. Players who playing on B should be able to teleport to warp which set at A. Also they should be able to teleport to player that stands on another server. Set homes should be compatible too.
All data can be stored in MySQL.

**Comment by colue** [comment]
i think this is not needed when you can have another plugin do it or you can use multiverse and setup worlds that homes will still be there.

**Comment by DevBD1** [comment]
You misunderstood. This feature request is not about "multi worlds", it is about Bungee servers?
CMI already has multiworld support, you don't need multiverse for that. :D

**Comment by LoToRTv** [comment]
rly good idea +1

**Comment by derMiepz** [comment]
Yes please add it. I can't understand why this isn't a thing already as portals are already implemented and working just fine.

**Comment by derMiepz** [comment]
Seems like this won't happen anytime... "soon"

**Comment by mrfloris** [comment]
I am still poking Zrips frequently about pushing bungeecord/velocity support.
